README.md

```diff
@@ -2,18 +2,18 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio
```

Comment:
- [ ] I would recommend you write a proper description for your website which will make your repo look professional. By doing so, you are also selling yourself to your future recruiters 😄.